### DISCUSSION TITLE
Подсчет совпадений значений полей одного столбца.

### DISCUSSION THREAD

**Discussion by gddno** [totumonline/totum-ru-issues-and-discussions#147] [Как это сделать?] [answered]
Требуется осуществить поиск всех полей в столбце "А", значение в которых совпадает с выбранным полем. Затем посчитать их количество и вывести результат в поле столбца "Количество совпадений по столбцу А"

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Несмотря на простоту задачи — она требует комплексного подхода:

### [Видео на YouTube](https://youtu.be/Pt4ydTG0v9Y)

Этот пример для загрузки: [download link](https://raw.githubusercontent.com/totumonline/ttm-samples/main/samples/totum_sample_12.json.gz.ttm)

Как загрузить: https://github.com/totumonline/ttm-samples

**Reply by ttm-alexeypolunin** [reply]
### ВАЖНОЕ ЗАМЕЧАНИЕ! — ☝️ ЭТО ДЛЯ ПРОСТЫХ ТАБЛИЦ И ЦИКЛОВ

В Расчетных и Временных при вычислении значения не нужны коды действия с `recalculate`, но колонка с вычисляемым значением должна быть после всех участвующих полей и колонки участвующих полей должны быть вычислены **полностью!**

Поэтому в таблицах с Табличной единицей пересчета надо изменить тип расчета со стандартного строчного на поколоночный. Смотреть здесь: https://ru.docs.totum.online/calculation-sequence#table-full-column
